packages/mobile/src/screens/favorites-screen/AlbumsTab.tsx

```diff
@@ -60,7 +57,7 @@ export const AlbumsTab = () => {
           <CollectionList
             listKey='favorites-albums'
             scrollEnabled={false}
-            collection={userAlbums ?? []}
+            collection={(userAlbums as UserCollection[]) ?? []}
```

Comment:
Having to cast this unnecessarily, since collection list assumes UserCollection, even though it only needs that when it's a "user" type card. we could probably improve this typing after reloaded